proc/threads.go

```diff
@@ -358,12 +453,18 @@ func (thread *Thread) SetCurrentBreakpoint() error {
 	return nil
 }
 
+func (thread *Thread) resetBreakpoint() {
```

Comment:
nit: rename to something like: `clearBreakpointState` or something along those lines to make it clear it's basically just zeroing out it's own state.